pwiz/data/vendor_readers/ABI/T2D/Reader_ABI_T2D.hpp

```diff
@@ -62,6 +62,7 @@ class PWIZ_API_DECL Reader_ABI_T2D : public Reader
     }
 
     virtual const char * getType() const {return "ABSciex T2D";}

```

Comment:
Drop AB

Reply:
Not sure about this one in particular. T2D is used for the old ABI TOF-TOF 4700/4800/5800. Maybe it should actually be "ABI T2D" instead.

Reply:
Okay. Anything that is ABSciex needs to become just Sciex or SCIEX, but if it is an old system pre-ABSciex, then maybe it is okay to make it ABI. Is the system supported by anyone? Is there a web page for it? If so, who supports it? What does the web page call it?

Reply:
It looks like the 4700 and 4800 went to Thermo with the Applied Biosystems acquisition (https://assets.thermofisher.com/TFS-Assets/LSG/manuals/cms_041368.pdf). Meanwhile Sciex further developed the TOF-TOF into the 5800, which is no longer in production but has a page on their site (https://sciex.com/products/mass-spectrometers/tof/tof-systems/tof/tof-5800-system). They all use the same Oracle database backend and T2D file export. Murky waters indeed. "AB/Sciex" might be ok (either/or), but not "AB Sciex" (like the name of a merged company)

Reply:
Okay. Agreed. "AB/Sciex" sounds like the way to go. Thanks for the detailed look.